src/main/java/com/google/sps/servlets/ImageFormServlet.java

```diff
@@ -0,0 +1,39 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.UUID;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.MultipartConfig;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.Part;
+
+@WebServlet("/upload")
```

Comment:
This is good but another approach to consider is making the servlet more general and creating CRUD/REST operations to handle each of the different types. So you'd have a `doPost` for the upload and a `doGet` for the list.

I think this current approach is good with 4 people working on the same codebase though so I'm actually going to suggest leaving it as-is and letting the list servlet do its own thing separately.